profiles/urls.py

```diff
@@ -2,6 +2,5 @@
 from . import views
 
 urlpatterns = [
-    # URL-шлях для сторінки профілю користувача
-    path('profile/', views.user_profile, name='user_profile'),
+    path('user_profile/', views.user_profile, name='user_profile'),
```

Comment:
this is not REST approach. Makes sure that you make CRUD API interface with rest URI names.
follow best practices